docs/en/04_Changelogs/4.9.0.md

```diff
@@ -0,0 +1,15 @@
+# 4.9.0 (Unreleased)
```

Comment:
4.9.0 is now out, so we probably need to target `4.9` branch to make this into `4.9.1` and update the changelog accordingly, is that right @maxime-rainville?

Reply:
I think this can be treated as a bugfix — it fixes broken/unexpected behaviour, so I think it doesn't need to be treated as a new feature.